python/ml4ir/config/default_model_config.yaml

```diff
@@ -0,0 +1,22 @@
+architecture_key: dnn
```

Comment:
this is great.  Exactly what I was looking for in terms of externalizing simple arch config.